reps_event_app/lib/ui/splashScreen.dart

```diff
@@ -47,7 +49,7 @@ class _SplashScreenPageState extends State<SplashScreenPage>
 
   navigationToHomePage() {
     Navigator.pushReplacement(
-        context, MaterialPageRoute(builder: (context) => Events()));
+        context, MaterialPageRoute(builder: (context) => Reps()));
```

Comment:
`Events` was the homepage if you will navigate to `Reps` than `Reps` will be homepage.
So replace `Reps` with `Events`